azure-application-insights-spring-boot-starter/src/main/java/com/microsoft/applicationinsights/boot/ApplicationInsightsProperties.java

```diff
@@ -201,7 +202,7 @@ public void setInProcess(InProcess inProcess) {
       private boolean throttling = true;
 
       /** Sets the size of maximum instant retries without delay */
-      private int maxInstantRetry = InProcessTelemetryChannel.DEFAULT_MAX_INSTANT_RETRY;
+      private int maxInstantRetry = ATelemetryChannel.DEFAULT_MAX_INSTANT_RETRY;
```

Comment:
Why the name change?

Reply:
I pulled up a lot of boiler-plate Channel code into a base class to share between InProcessTelemetryChannel and LocalForwarderTelemetryChannel. The constants were part of it.